flight/Modules/Stabilization/stabilization.c

```diff
@@ -219,6 +220,15 @@ static void stabilizationTask(void* parameters)
 		dT = PIOS_DELAY_DiffuS(timeval) * 1.0e-6f;
 		timeval = PIOS_DELAY_GetRaw();
 		
+		// exponential moving averaging (EMA) of dT to reduce jitter; ~200points
+		// to have more or less equivalent noise reduction to a normal N point moving averaging:  alpha = 2 / (N + 1)
+		dT_filtered = 0.01f * dT + (1.0f - 0.01f) * dT_filtered;
+
+		if (gyro_alpha_temp  < 0.0001f) // not trusting this to resolve to 0
+			gyro_alpha = 0;   // Gyro LPF should be disabled
+		else
+			gyro_alpha = expf(-gyro_alpha_temp * dT_filtered);
```

Comment:
Maybe it could be calculated once after the FC has been running for a few seconds. Re-calculating over and over again is kind of silly as the gyro sampling rate is constant (and so `dT_filtered` will converge to a constant value quite quickly).